libs/ledger-live-common/src/hw/actions/implementations.ts

```diff
@@ -147,17 +148,16 @@ const pollingImplementation: Implementation = <SpecificType, GenericRequestType>
 
             debounce((event: EmittedEvent<SpecificType> | ImplementationEvent) => {
               if (event.type === "error" && "error" in event) {
+                const {
+                  retryErrorsWhitelist = [DisconnectedDevice, DisconnectedDeviceDuringOperation],
+                } = params;
```

Comment:
Maybe can you add a comment on top of `pollingImplementation` to explain what `retryErrorsWhiteList` does and that it has a default value pls ? (don't explain all the other fields of `params` no worries - just the one you added)

And i feel people try to avoid saying white/black list now. Maybe another name can be `retryableDisconnectedErrors` or `retryableWithDelayDisconnectedErrors` (because also those errors are being delayed with a `reconnectWaitTime` so they are kind of associated to disconnection errors)

Reply:
the code is quite hard to read, i initially thought it was a params deconstruction. 

I would suggest you create a `defaultErrors` somewhere.

also, generally speaking i think it would be more readable to do

```js
const errors = params.retryableWithDelayDisconnectedErrors || defaultErrors
```